ddcommon/src/tag.rs

```diff
@@ -32,9 +89,8 @@ impl Display for Tag {
 }
 
 impl Tag {
-    /// It's recommended to use Tag::new when possible, as tags that are in
-    /// the <KEY>:<VALUE> format are preferred.
-    pub fn from_value<'a, IntoCow>(chunk: IntoCow) -> Result<Self, Cow<'static, str>>
+    /// Validates a tag.
+    fn from_value<'a, IntoCow>(chunk: IntoCow) -> Result<Self, Cow<'static, str>>
```

Comment:
Should this be `try_from`?

Reply:
That would make it a public since it's a trait impl. It's an implementation detail used by `parse_tags` and `new`.